src/userAgent.js

```diff
@@ -0,0 +1,93 @@
+/**
+ * @license
+ * This file is part of the Game Closure SDK.
+ *
+ * The Game Closure SDK is free software: you can redistribute it and/or modify
+ * it under the terms of the Mozilla Public License v. 2.0 as published by Mozilla.
+
+ * The Game Closure SDK is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * Mozilla Public License v. 2.0 for more details.
+
+ * You should have received a copy of the Mozilla Public License v. 2.0
+ * along with the Game Closure SDK.  If not, see <http://mozilla.org/MPL/2.0/>.
+ */
+
+ /**
+ * @module userAgent
+ *
+ * Uses navigator.userAgent and retrieves information about app runtime, device, OS version,
+ * simulator, and browser version.
+ *
+ */
+
+var ua = navigator && navigator.userAgent;
+exports.USER_AGENT = ua;
```

Comment:
Let's group all the exports at the bottom in one chunk so it's easy to see everything together